internal/builtin/jwt.go

```diff
@@ -0,0 +1,131 @@
+package builtin
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/lestrrat-go/jwx/v3/jwa"
+	"github.com/lestrrat-go/jwx/v3/jwt"
+)
+
+type Jwt struct {
+}
+
+func NewJwt() *Jwt {
+	return &Jwt{}
+}
+
+type JWTOptions struct {
+	Secret        string         `json:"secret"`         // Required
+	Algorithm     string         `json:"algorithm"`      // Optional: HS256, HS384, HS512 (default: HS256)
+	Subject       string         `json:"subject"`        // Optional: sub claim
+	Audience      []string       `json:"audience"`       // Optional: aud claim
+	Issuer        string         `json:"issuer"`         // Optional: iss claim
+	ID            string         `json:"id"`             // Optional: jti claim
+	ExpiresIn     string         `json:"expires_in"`     // Optional: duration like "1h", "30m"
+	NotBefore     string         `json:"not_before"`     // Optional: duration like "5m"
+	PrivateClaims map[string]any `json:"private_claims"` // Optional: private claims
+}
+
+func (j *Jwt) Sign(opts map[string]any) string {
+	var opt *JWTOptions
+	b, err := json.Marshal(opts)
+	if err != nil {
```

Comment:
```suggestion
func (j *Jwt) Sign(opts map[string]any) string {
```

Please take a look at the results of `make lint`.